soccer/gameplay/plays/offense/cross_kick.py

```diff
@@ -0,0 +1,272 @@
+        self.add_transition(CrossKick.State.crossing,
+        	                CrossKick.State.collecting, lambda: False, 'should not restart when in scoring position')
+        self.add_transition(CrossKick.State.crossingLeft,
+        	                CrossKick.State.collecting, lambda: False, 'should not restart when in scoring position')
+        self.add_transition(CrossKick.State.recrossing,
+        	                CrossKick.State.collecting, lambda: False, 'should not restart when in scoring position')
+        self.add_transition(CrossKick.State.recrossing, 
+                            CrossKick.State.finishing, lambda: ready_to_finish & (self.subbehavior_with_name('recapture').state == behavior.Behavior.State.completed), 
+                            'scoring position')
+        self.add_transition(CrossKick.State.recrossing, 
+                            CrossKick.State.transitioning, lambda: (not ready_to_finish) & (self.subbehavior_with_name('recapture').state == behavior.Behavior.State.completed), 
+                            'not in scoring position')
+        #self.add_transition(CrossKick.State.finishing, 
+        	                #behavior.Behavior.State.completed, lambda: self.subbehavior_with_name('shoot').state == behavior.Behavior.State.completed, 'shot my shot')
+        '''
+        self.add_transition(CrossKick.State.transitioning, 
+                            CrossKick.State.crossing, lambda: self.subbehavior_with_name('capture').state == behavior.Behavior.State.completed, 
+                            'robots in position')'''
+        '''self.add_transition(CrossKick.State.transitioning, 
+        	                CrossKick.State.collecting, lambda: self.subbehavior_with_name('pass to flank').state == behavior.Behavior.State.failed, 'restart')'''
+        '''self.add_transition(CrossKick.State.crossing, 
+                            CrossKick.State.finishing, lambda: self.subbehavior_with_name('capture cross').state == behavior.Behavior.State.completed,
+                            'finishing the play')'''
+        '''self.add_transition(CrossKick.State.crossing, 
+        	                CrossKick.State.collecting, lambda: self.subbehavior_with_name('box pass').state == behavior.Behavior.State.failed, 'restart')'''
+        '''self.add_transition(CrossKick.State.finishing, 
+        	                behavior.Behavior.State.completed, lambda: self.subbehavior_with_name('shoot').state == behavior.Behavior.State.completed, 'shot my shot')'''
+
+    def on_enter_collecting(self):
+        self.remove_all_subbehaviors()
+
+        capture = skills.capture.Capture()
+        self.add_subbehavior(capture, 'capture', required=True) 
+
+    def on_exit_collecting(self):
+        #self.captureRobot = self.subbehavior_with_name('capture')
+        self.remove_all_subbehaviors()
+
+    
+    def on_enter_transitioning(self): 
```

Comment:
Should you rename this to on_enter transitioning_right? Just to give more description of what you're doing?